docs/runbooks/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,36 @@
+# KubeVirtDeprecatedAPIsRequested
+<!-- Edited by fmatouschek, May 2023-->
+
+## Meaning
+
+This alert fires when a deprecated KubeVirt API was requested.
+
+## Impact
+
+Usage of deprecated APIs is not recommended because they will be removed in a future release.
+
+## Diagnosis
+
+Check the actual deprecated API usage by using a PromQL query:
+
+```
+kubevirt_api_request_deprecated_total
```

Comment:
@0xFelix I beileve you should ask to run the same query as in the alert:
```
sum by(resource,group,version) ((round(increase(kubevirt_api_request_deprecated_total{verb!~\"LIST|WATCH\"}[10m]))>0 and kubevirt_api_request_deprecated_total{verb!~\"LIST|WATCH\"} offset 10m) or (kubevirt_api_request_deprecated_total{verb!~\"LIST|WATCH\"} != 0 unless kubevirt_api_request_deprecated_total{verb!~\"LIST|WATCH\"} offset 10m))
```
and state that they can update the `10m` to the time period they want to inspect